### PR TITLE
Image and video picker component.

### PR DESCRIPTION
**Pull request by suraj-webkul**
## Description
Image and video picker component.

### CODE REVIEW

File: packages/Webkul/Shop/src/Http/Resources/ProductReviewResource.php [hunk 1]
```diff
@@ -14,12 +14,19 @@ class ProductReviewResource extends JsonResource
      */
     public function toArray($request)
     {
+        $images = $this->images->map(function ($image) {
+            $image->url = $image->url;
+        
+            return $image;
+        });
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Http/Resources/ProductReviewResource.php:21]: Remove this logic. And add the `url` key to the appends property.

File: packages/Webkul/Product/src/Repositories/ProductReviewAttachmentRepository.php [hunk 2]
```diff
@@ -4,19 +4,15 @@
...
      * Specify Model class name
-     *
-     * @return string
      */
-
-    function model(): string
+    public function model(): string
     {
-        return ProductReviewAttachment::class;
+        return 'Webkul\Product\Contracts\ProductReviewAttachment';
```
Comment (devansh-webkul) [inline packages/Webkul/Product/src/Repositories/ProductReviewAttachmentRepository.php:15]: Revert this..

**Review by devansh-webkul — CHANGES_REQUESTED** [review]
Please done the requested changes.

File: packages/Webkul/Shop/src/Http/Resources/ProductReviewResource.php (hunk 1 above)
Comment (suraj-webkul) [reply packages/Webkul/Shop/src/Http/Resources/ProductReviewResource.php:21]: done

File: packages/Webkul/Product/src/Repositories/ProductReviewAttachmentRepository.php (hunk 2 above)
Comment (suraj-webkul) [reply packages/Webkul/Product/src/Repositories/ProductReviewAttachmentRepository.php:15]: done

**Review by devansh-webkul — APPROVED** [review]

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**